inginious/frontend/pages/api/_api_page.py

```diff
@@ -101,17 +101,20 @@ class APIAuthenticatedPage(APIPage):
         A wrapper for pages that needs authentication. Automatically checks that the client is authenticated and returns "403 Forbidden" if it's
         not the case.
     """
-
     def _handle_api(self, handler, handler_args, handler_kwargs):
         return APIPage._handle_api(self, (lambda *args, **kwargs: self._verify_authentication(handler, args, kwargs)), handler_args, handler_kwargs)
 
     def _verify_authentication(self, handler, args, kwargs):
-        """ Verify that the user is authenticated """
-        if not self.user_manager.session_logged_in():
+        """ Verify that request have a http authorization bearer and that token match with a user."""
+        if 'Authorization' in request.headers:
+            auth_header = request.headers.get('Authorization')
+            token = auth_header.replace('Bearer ', '')
+            if self.user_manager.get_userinfo_from_apikey(token) is None:
```

Comment:
I'm wondering, at this stage of implementation, if it wouldn't be more interesting to use the Basic Authorization as it is done with WebDAV. Indeed, as the API key implementation is currently very straightforward, asking for the username would add a layer of complexity. Besides, this would be less confusing as the Bearer method is designed for tokens that are very more complex, encrypted, objects.

@nrybowski any thoughts on this ?